app/main.py

```diff
@@ -1,8 +1,66 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+
+        self.brand = brand
+
+        if comfort_class < 1 or comfort_class > 7:
```

Comment:
Well done! This will validate data on object creation. Later in the course we will study how to do it even more pythonic :)